README.md

```diff
@@ -1,109 +1,31 @@
 ![cf](https://i.imgur.com/7v5ASc8.png) Lab 07: Vanilla HTTP Server
```

Comment:
generally in readme, you should tell people how to use your application! eg- what to type in the command line to get your cow to talk